docs/standard/exceptions/how-to-use-the-try-catch-block-to-catch-exceptions.md

```diff
@@ -18,8 +19,9 @@ Place any code statements that might raise or throw an exception in a `try` bloc
 
 In the following example, a <xref:System.IO.StreamReader> opens a file called *data.txt* and retrieves a line from the file. Since the code might throw any of three exceptions, it's placed in a `try` block. Three `catch` blocks catch the exceptions and handle them by displaying the results to the console.
 
-[!code-csharp[CatchException#3](~/samples/snippets/csharp/VS_Snippets_CLR/CatchException/CS/catchexception2.cs#3)]
-[!code-vb[CatchException#3](~/samples/snippets/visualbasic/VS_Snippets_CLR/CatchException/VB/catchexception2.vb#3)]
+:::code language="cpp" source="./snippets/how-to-use-the-try-catch-block-to-catch-exceptions/cpp/catchexception.cpp" id="Snippet3":::
```

Comment:
Thank you for adding C++ here, much appreciated.

Reply:
I've seen it while preparing the move of the other page samples.
I also left some unused files in the samples folder, I did not find any usage of them, but did not want to delete them anyway.